src/SqlUtils.ts

```diff
@@ -4,39 +4,54 @@ import * as mssql from "mssql";
 import Constants from "./Constants";
 import SqlConnectionConfig from "./SqlConnectionConfig";
 
+export interface ConnectionResult {
+    success: boolean,                   // True if connection succeeds, false otherwise
```

Comment:
use js doc comments so that they show up when hovering over these properties when they're used in other places in the code

Reply:
Updated